packages/amplify-frontend-javascript/lib/frontend-config-creator.js

```diff
@@ -339,7 +329,8 @@ function getAPIGWConfig(apigwResources, projectRegion, configOutput) {
   };
 
   for (let i = 0; i < apigwResources.length; i += 1) {
-    if (apigwResources[i].output.ApiName && apigwResources[i].output.RootUrl) { // only REST endpoints contains this information
+    if (apigwResources[i].output.ApiName && apigwResources[i].output.RootUrl) {
```

Comment:
nit: this for loop can be simplified (I didn't test this snippet)

```ts
apigwConfig.aws_ckoud_logic_custom = apigwResources
  .filter(r => r.output.ApiName && r.output.RootUrl)
  .map(r => {
    return {
        name: r.output.ApiName,
        endpoint: r.output.RootUrl,
        region: projectRegion,
    }
  });
return apigwConfig;
```

Reply:
Thanks!! , works fine , tested it on my end